galsim/wfirst/__init__.py

```diff
@@ -0,0 +1,188 @@
+# Copyright (c) 2012-2014 by the GalSim developers team on GitHub
+# https://github.com/GalSim-developers
+#
+# This file is part of GalSim: The modular galaxy image simulation toolkit.
+# https://github.com/GalSim-developers/GalSim
+#
+# GalSim is free software: redistribution and use in source and binary forms,
+# with or without modification, are permitted provided that the following
+# conditions are met:
+#
+# 1. Redistributions of source code must retain the above copyright notice, this
+#    list of conditions, and the disclaimer given in the accompanying LICENSE
+#    file.
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions, and the disclaimer given in the documentation
+#    and/or other materials provided with the distribution.
+#
+"""
+The galsim.wfirst module, containing information GalSim needs to simulate images for the WFIRST-AFTA
+project.
+
+This module contains numbers and routines for the WFIRST-AFTA project.  There is also a demo
+illustrating the use of most of this functionality, demo13.py.  Some of the parameters below relate
+to the entire wide-field imager.  Others, especially the return values of the functions to get the
+PSF and WCS, are specific to each SCA (Sensor Chip Assembly, the equivalent of a chip for an optical
+CCD) and therefore are indexed based on the SCA.  All SCA-related arrays are 1-indexed, i.e., the
+entry with index 0 is None and the entries from 1 to n_sca are the relevant ones.  This is
+consistent with diagrams and so on provided by the WFIRST project, which are 1-indexed.
+
+Currently, the module includes the following numbers:
+
+    gain - The gain for all SCAs (sensor chip assemblies) is expected to be the same, so this is a
+           single value rather than a list of values.
+
+    pixel_scale - The pixel scale in units of arcsec/pixel.
```

Comment:
Probably ought to be described as approximate, since the pixel scale will vary slightly across the detectors.